components/Work/WorkPosts.tsx

```diff
@@ -0,0 +1,61 @@
+import Post from "../general/Post/Post";
+import styles from "./WorkPosts.module.css";
+import { PostProps } from "../general/Post/Post";
+
+const WorkPosts = () => {
+  const posts: PostProps[] = [
+    {
+      sectionName: "dancing\nstars",
+      desOne: "content",
+      desTwo: "video",
+      desThree: "design",
```

Comment:
Think about naming again. Naming is very important especially when you dealing with a group of devs.